ChangeLog.md

```diff
@@ -33,12 +42,39 @@
 * Settings tab layout has been redesigned to support longer texts
 * Added button to start map editor directly from Launcher
 * Simplified game starting flow from online lobby
+* Mod description will now show list of languages supported by mod
+* Launcher now uses separate mod repository from vcmi-1.1 version to prevent mod updates to unsupported versions

```

Comment:
> to prevent mod updates to unsupported versions
1) I thought it works well... actually launcher shall verify version before install it
2) repo links are stored locally, so if user override it earlier (very possible) - new repo won't be visible (if logic wasn't changed intentionally)
3)
```suggestion
* Launcher now uses separate mod repository from vcmi-1.1 version to prevent mod updates to unsupported versions
* Adjustable size of mod list and mod details sub-windows
```

Reply:
> I thought it works well... actually launcher shall verify version before install it (iirc, that was the behavior I made)
1) Very few mods actively maintain & update that field
2) Since we only provide single download link users who do not update to 1.2 will quickly get unusable repo (with half of mods not installable due to no longer being compatible). 
3) All players who use daily builds must be told to use separate 'beta' repo. Seen multiple players who use daily builds and can't find working repo link.

> repo links are stored locally, so if user override it earlier (very possible) - new repo won't be visible (if logic wasn't changed intentionally)

I doubt that many users have changed that. And if they do - at the very least they are those who are aware of repo list existance and can change it to new url.

> 3.

Sure, but I think it was adjustable in 1.1 and only changed in 1.2? So no change as far as 1.1 players are concerned?